metric-anomaly-data-model/src/main/proto/org/hypertrace/alert/engine/eventcondition/config/service/v1/eval_rule.proto

```diff
@@ -2,17 +2,23 @@ syntax = "proto3";
 
 option java_multiple_files = true;
 
-package org.hypertrace.alert.engine.metric.anomlay.rule.config.service.v1;
+package org.hypertrace.alert.engine.eventcondition.config.service.v1;
 
-import "org/hypertrace/metric/anomaly/rule/config/service/v1/alert_rule_enum.proto";
+import "org/hypertrace/alert/engine/eventcondition/config/service/v1/eval_rule_enums.proto";
 
-message MetricAnomalyAlertRule {
+
+message MetricAnomalyEventCondition {
   string name = 1;
   repeated Label labels = 2;
+  MetricAnomalyEvalRule eval_rule = 3;
+  NotificationCondition notification_condition = 4;
+}
+
+message MetricAnomalyEvalRule {
   MetricSelection metric_selection = 3;
```

Comment:
Filter need not be repeated in the MetricSelection proto (existing code)

Reply:
didn't follow this - is there another filter in the message tree elsewhere?

Reply:
No, there is no other filter somewhere else. 
@rish691  I also didn't get it.

Reply:
@kotharironak refer to this comment https://github.com/hypertrace/hypertrace-alert-engine/pull/7#discussion_r657087625

Reply:
Ah! I see now. Accepted the changes.